README.md

```diff
@@ -66,6 +66,13 @@ remotes::install_github("mlr-org/mlr3learners", dependencies = TRUE)
 | [regr.svm](https://mlr3learners.mlr-org.com/reference/mlr_learners_regr.svm.html)             | SVM                         | [e1071](https://cran.r-project.org/package=e1071)             |
 | [regr.xgboost](https://mlr3learners.mlr-org.com/reference/mlr_learners_regr.xgboost.html)     | Gradient Boosting           | [xgboost](https://cran.r-project.org/package=xgboost)         |
 
+## Survival Learners
+
+| ID                                                           | Learner                     | Package                                             |
+| :----------------------------------------------------------- | :-------------------------- | :-------------------------------------------------- |
+| [surv.cv_glmnet](https://mlr3learners.mlr-org.com/reference/mlr_learners_surv.cv_glmnet.html) | Penalized Linear Regression | [glmnet](https://cran.r-project.org/package=glmnet) |
+| [surv.glmnet](https://mlr3learners.mlr-org.com/reference/mlr_learners_surv.glmnet.html) | Penalized Linear Regression | [glmnet](https://cran.r-project.org/package=glmnet) |
+
```

Comment:
I don't think we should add this category. It also does not fit into the existing category logic.

Reply:
not sure what you mean by existing category logic as it's identical to regr and classif, but no prob i've deleted it :)